core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -20,26 +21,28 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.duration.FiniteDuration
 
 object RawlsApiServiceActor {
-  def props(workspaceServiceConstructor: UserInfo => WorkspaceService, userServiceConstructor: UserInfo => UserService, genomicsServiceConstructor: UserInfo => GenomicsService, statisticsServiceConstructor: UserInfo => StatisticsService, statusServiceConstructor: () => StatusService, appVersion: ApplicationVersion, googleClientId: String, submissionTimeout: FiniteDuration)(implicit executionContext: ExecutionContext): Props = {
-    Props(new RawlsApiServiceActor(workspaceServiceConstructor, userServiceConstructor, genomicsServiceConstructor, statisticsServiceConstructor, statusServiceConstructor, appVersion, googleClientId, submissionTimeout))
+  def props(workspaceServiceConstructor: UserInfo => WorkspaceService, userServiceConstructor: UserInfo => UserService, genomicsServiceConstructor: UserInfo => GenomicsService, statisticsServiceConstructor: UserInfo => StatisticsService, statusServiceConstructor: () => StatusService, appVersion: ApplicationVersion, googleClientId: String, submissionTimeout: FiniteDuration, rawlsMetricBaseName: String)(implicit executionContext: ExecutionContext): Props = {
+    Props(new RawlsApiServiceActor(workspaceServiceConstructor, userServiceConstructor, genomicsServiceConstructor, statisticsServiceConstructor, statusServiceConstructor, appVersion, googleClientId, submissionTimeout, rawlsMetricBaseName))
   }
 }
 
-class RawlsApiServiceActor(val workspaceServiceConstructor: UserInfo => WorkspaceService, val userServiceConstructor: UserInfo => UserService, val genomicsServiceConstructor: UserInfo => GenomicsService, val statisticsServiceConstructor: UserInfo => StatisticsService, val statusServiceConstructor: () => StatusService, val appVersion: ApplicationVersion, val googleClientId: String, val submissionTimeout: FiniteDuration)(implicit val executionContext: ExecutionContext) extends Actor
+class RawlsApiServiceActor(val workspaceServiceConstructor: UserInfo => WorkspaceService, val userServiceConstructor: UserInfo => UserService, val genomicsServiceConstructor: UserInfo => GenomicsService, val statisticsServiceConstructor: UserInfo => StatisticsService, val statusServiceConstructor: () => StatusService, val appVersion: ApplicationVersion, val googleClientId: String, val submissionTimeout: FiniteDuration, val rawlsMetricBaseName: String)(implicit val executionContext: ExecutionContext) extends Actor
```

Comment:
It just indicates you're overriding the val from a base class, like Java `@Override`. I think it's a compiler warning otherwise?